modules/client/src/Wallet.ts

```diff
@@ -5,7 +5,7 @@ import { IConnextChannelOptions } from './Connext'
 import {
   objMapPromise,
   TransactionRequest,
-  TransactionResponse,
+  TransactionResponse
```

Comment:
We thought KT's vote for trailing commas was the tie breaker but apparently this debate is tied up again 😅